deploy/terraform-datacommons-website/examples/setup/variables.tfvars

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ # The follow fields must be filled out before running the setup script.
+
+# GCP project id where you wish to set up the Data Commons instance in.
+# Ideally this is a new GCP project dedicated to Data Commons.
+#
+# Please consult the doc below on how to create one,
+# or reach out to your GCP org admin to have one created.
+# https://developers.google.com/workspace/guides/create-project
+project_id="Replace me"
+
+
+# The setup script will create a new domain that will host the Data Commons
+# web application. As a part of the standard domain registration process,
+# the script requires the domain you wish to create, and the contact information
+# associated with the domain. The information is used for registering the domain,
+# and will not be publicly available.
+
+# The domain you wish to create. During the installation step, the Data Commons
+# web application will be installed under this domain.
+#
+# Please not that this domain must be available, or else the script will fail.
+# To check if the domain is available, please search it up using the link below.
+# https://domains.google/
+# If you see a green checkmark in the search results, it means that is available.
+# Please also fill out the associated price for the domain you see on the search
+# result. It must be an exact match in the format of "<amount 2 decimal> <currency".
+# Ex: "12.00 USD"
+dc_website_domain="Replace me"
```

Comment:
For the fields below, how much of them can have default value?

For example, I can think if dc_website_domain is empty, then the script should set a default one like "datacommons.website.<project_id>.com", with a "domain_yearly_price" of 12?